docs/gateway/s3.md

```diff
@@ -33,15 +33,15 @@ As a prerequisite to run Minio S3 gateway on an AWS S3 compatible service, you n
 docker run -p 9000:9000 --name minio-s3 \
  -e "MINIO_ACCESS_KEY=access_key" \
  -e "MINIO_SECRET_KEY=secret_key" \
- minio/minio gateway s3 --address https://s3_compatible_service_endpoint:port
+ minio/minio gateway s3 https://s3_compatible_service_endpoint:port
 ```
 
 ### Using Binary
 
 ```
 export MINIO_ACCESS_KEY=access_key
 export MINIO_SECRET_KEY=secret_key
-minio gateway s3 --address https://s3_compatible_service_endpoint:port
+minio gateway s3 https://s3_compatible_service_endpoint:port
```

Comment:
what if the custom s3 endpoint is `http` instead of `https` should we specify that too?